File: app/src/main/java/mb00/android/codehub/ui/settings/inject/SettingsActivityModule.java
package mb00.android.codehub.ui.settings.inject;

import dagger.Module;
import dagger.Provides;
import mb00.android.codehub.ui.settings.viewmodel.SettingsViewModel;


@Module
public class SettingsActivityModule {

    @Provides
    SettingsViewModel provideSettingsViewModel() {
        return new SettingsViewModel();
    }

}

File: app/src/main/java/mb00/android/codehub/ui/settings/inject/SettingsActivityModule.kt
package mb00.android.codehub.ui.settings.inject

import dagger.Module
import dagger.Provides
import mb00.android.codehub.ui.settings.viewmodel.SettingsViewModel


@Module
class SettingsActivityModule {

    @Provides
    internal fun provideSettingsViewModel(): SettingsViewModel {
        return SettingsViewModel()
    }

}
